packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -987,7 +987,7 @@ class Auth {
      *
      */
     async authorizeIDP(parameters: AuthorizeIDPParams) {
-        const redirectURI = this.redirectURI
+        const redirectURI = parameters.redirectURI || this.redirectURI
```

Comment:
Similar to `loginIDPUser`, if a user passes in a `redirectURI`, we want to prioritize using this value